www/templates/control/main-control.html

```diff
@@ -29,20 +29,33 @@
       </label>
     </div>
     <div class="control-list-item">
-      <div class="control-list-text">Force sync</div>
-      <div ng-click="forceSync()" ng-style="getIconButtonStyle('D6C780')"><i class="ion-android-sync" ng-style="getIconStyle()"></i></div>
-    </div>   
+      <div class="control-list-text">Low accuracy</div>
+      <label ng-click="willUseLowAccuracy()" class="toggle control-list-toggle">
```

Comment:
you should be able to use ng-model instead. I had this as feedback for some other toggle in the control screen as well.